build/templates/build-and-run-az-func-container.yml

```diff
@@ -0,0 +1,62 @@
+parameters:
+  containerName: ''
+  projectName: ''
+  imageName: ''
+  imageTag: ''
+  port: ''
```

Comment:
The new parameter format can help you with validation - https://docs.microsoft.com/en-us/azure/devops/pipelines/yaml-schema?view=azure-devops&tabs=schema%2Cparameter-schema#parameters

But that might be a dedicated PR

Reply:
Added one here: https://github.com/arcus-azure/arcus/issues/141